src/layers/MultiscaleImageLayer/MultiscaleImageLayer.js

```diff
@@ -95,7 +96,10 @@ export default class MultiscaleImageLayer extends CompositeLayer {
       maxRequests,
       onClick,
       modelMatrix,
-      transparentColor
+      transparentColor,
+      excludeBackground,
```

Comment:
Add `excludeBackground` to the `defaultProps` and docs?

Reply:
Done.